source/delayed_capture/index.html.md.erb

```diff
@@ -18,27 +18,36 @@ You cannot use delayed capture for:
 * payments made with [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/)
 * [recurring payments](/recurring_payments)
 
-To use this feature, include `"delayed_capture": true` in the body of a [Create new payment](/making_payments/#creating-a-payment) request.
+To use this feature, include `"delayed_capture": true` in the body of your [request to create a new payment](/making_payments/#creating-a-payment).
 
-The user experience matches the current payment flow. Once your user selects
-__Confirm__ on the __Confirm your details__ page:
+This example request creates a £50 parking fine that the service can capture at a later point because it includes the `delayed_capture` parameter:
 
-- the [payment status](/api_reference/#payment-status-lifecycle) will change to `capturable`
-- your service can make a request to the
-`POST /v1/payments/{PAYMENT_ID}/capture` endpoint to send a delayed capture
-request
+`POST /v1/payments`
 
-Your service must send the delayed capture request within 120 hours (5 days) of creating the payment, regardless of how long your user takes to complete the payment flow. If you do not, [the payment will expire](/payment_flow/#6-your-user-confirms-payment).
+```json
+{
+  "amount": 5000,
+  "reference" : "LE2504",
+  "description": "Pay a penalty charge notice (PCN)",
```

Comment:
```suggestion
  "description": "Pay a Penalty Charge Notice (PCN)",
```

Reply:
@ChristinaKyriakou I initially thought the same, but [the GOV.UK page for PCNs](https://www.gov.uk/parking-tickets) doesn't capitalise it.